app/main/utils/dto.py

```diff
@@ -21,3 +21,17 @@ class ClientDto:
         'name': fields.String(required=True, description="The Client name"),
         'secret': fields.String(required=True, description="The Client Secret"),
     })
+
+
+class EmployeeDto:
+    api = Namespace('employee', description="Employee related operations")
+    employee = api.model('employee', {
+        'first_name': fields.String(required=True, description="The employee's first name"),
+        'last_name': fields.String(required=True, description="The employee's last name"),
+        'email': fields.String(required=True, description="The employee's email"),
+    })
+    new_employee = api.model('new_employee', {
+        'first_name': fields.String(required=True, description="The employee's first name"),
+        'last_name': fields.String(required=True, description="The employee's last name"),
+        'email': fields.String(required=True, description="The employee's email"),
+    })
```

Comment:
Seems strange to duplicate here. New employee is still an employee. I'd use the same model unless there's a reason to have a separate one that I can't think of?